CHANGELOG.md

```diff
@@ -18,6 +19,18 @@
 - [v0.5.0](#v050---2020-08-06)
 <!-- END TOC -->
 
+-------------------------------------------------
+## v0.19.2 - 2022-03-17
+
+
+### Changed
+- Added the repo - "quay.io/jetstack/cert-manager-acmesolver" in allowrepo safeguard by default.
+- Backup operator namespaces can for example be added as veloro parameters to be able to back them up. 'alertmanager' is added as default in the workload cluster.
```

Comment:
This doesn't seem to be in this patch so it shouldn't be part of the changelog.

And there is no note of the fix for disabling `predictLinear`in the workload cluster.

Reply:
I removed the backup operator change. and added that we disabled predict linear in workload cluster.

Reply:
I meant to add it under "Fixed", since it's now fixed so you can disable it there. We don't really disable it by default.

Reply:
changed it.